plugin/pkg/scheduler/factory/factory.go

```diff
@@ -338,6 +338,11 @@ func (f *ConfigFactory) CreateFromConfig(policy schedulerapi.Policy) (*scheduler
 			}
 		}
 	}
+	// issue: https://github.com/kubernetes/kubernetes/issues/43845
+	// for backward compatibility if user set the value in policy file, then we use it
```

Comment:
Policy is the new location of `HardPodAffinitySymmetricWeight`, so backward compatibility is not addressed here, unless I am missing something.